apps/music/test/marionette/lib/playlisthelper.js

```diff
@@ -4,6 +4,10 @@
 
 var PlaylistHelper = {
 
+  mainTitle: function(element) {
+    return element.findElement('.list-main-title').text();
```

Comment:
I assume that the marionette text() call returns the textContent property of the element, or that it recursively traverses the element tree and returns the concatenation of the textContent properties of all descendants? Assuming that's true, then this seems fine. If it returns innerHTML, then it isn't what we need for this test, of course.